internal/functions/Invoke-ManagedComputerCommand.ps1

```diff
@@ -48,11 +48,45 @@ function Invoke-ManagedComputerCommand {
 
     [scriptblock]$setupScriptBlock = {
         $ipaddr = $args[$args.GetUpperBound(0)]
-
-        # Just in case we go remote, ensure the assembly is loaded
-        [void][System.Reflection.Assembly]::LoadWithPartialName('Microsoft.SqlServer.SqlWmiManagement')
-        $wmi = New-Object Microsoft.SqlServer.Management.Smo.Wmi.ManagedComputer $ipaddr
-        $null = $wmi.Initialize()
+        
+        # Just in case we go remote, ensure the assembly is loaded. Wenn SQL Server 2014 and 13.0.0.0__89845dcd8080cc91 -> 
+        #[void][System.Reflection.Assembly]::LoadWithPartialName('Microsoft.SqlServer.SqlWmiManagement') - Failed!!!
+	    # Find right library
+		$Count = -1
+		$i = 0
+		$t = Get-ChildItem "C:\Windows\assembly\GAC_MSIL\Microsoft.SqlServer.SqlWmiManagement\*"|Sort Name -Descending
+		$t | Foreach{
```

Comment:
This needs to be formatted to our standard. If you use VS Code you can simply use the format document option from the command palette.

Reply:
Things like use of `foreach-object` instead of `foreach`. Most PowerShell keywords are lowercased like `if` and `else`. As well we use OTBS style formatting.

Reply:
Thank you for the review shawn 🙇

Reply:
I see that my changes not working. I will close PR.